composer.json

```diff
@@ -44,7 +49,8 @@
     "preferred-install": {
       "*": "dist"
     },
-    "sort-packages": true
+    "sort-packages": true,
+    "bin-dir": "bin/"
```

Comment:
is there any reason to do it ? As we are always using command runners (Makefile, GithubActions, ...), are you sure we need those bins in the root bin dir ?

Reply:
It doesn't really matter because they're ignored (only bin/console and user generated bin are commited). I like to have all the binaries inside the global bin/ directory. IT's messy when you try to manually launch commands and try to remember if you need to vendor/bin versus bin/

Reply:
As you are working on an already existing project, I'm not sure you'll always have what you like to have in an open source project with many other people. For my part I would prefer to have vendor related bin files in the vendors folder, and app related bin files in the root folder. And other contributor can prefer other things.

Some points :
- You PR is really valuable: it should be nice to use Github Actions in a github hosted project!
- We should keep in mind that our current GitlabCI config was doing exactly the same job in 10 lines instead of 128.
- Github actions are better than GitlabCI for a github project because of one point : GitlabCI cannot deal with Pull Request from another repository
- Adding phpcs and phpcbf is a really good idea, but maybe we should have added it in a dedicated PR in order to merge this one as soon as possible
- Removing code coverage is not a very good thing, but as the current coverage is about 7%, it's not so important
- Same thing for changing the php-cs-fixer config: imho, a dedicated PR would have been easier to review
- I don't see the point of using .phar file and another bin-dir if there is no need and if it's not something strongly recommended

=> Could you revert the not so useful changes like .phar and bin-dir ? We can keep everything else and see later for those points !

Reply:
Sure, I'll revert it.
Your logic is broken for bin/console:  it's a vendor one and is not inside the vendor ;)